src/command_modules/azure-cli-dla/HISTORY.rst

```diff
@@ -0,0 +1,21 @@
+.. :changelog:
+
+Release History
+===============
+0.0.1b3 (2017-03-22)
++++++++++++++++++++++
+
+* Rename and separate command modules for Data Lake Analytics and Data Lake Store
+* Remove datalake package
+* rename parameters for some catalog management sub groups
+* Fix support for the job list command to allow better filtering and ordering
+
```

Comment:
This history would start from afresh so I'd remove the older versions.

Or you can start azure-cli-dla from 0.0.1b1. Up to you.